### DISCUSSION TITLE
Can't log in with browser when app installed

### DISCUSSION THREAD

**Discussion by majora2007** [community/community#110452] [Mobile] [answered]
If you have the app installed and are trying to log into GitHub.com on your Firefox app on your phone, after logging in you are transfered into Mobile saying try again.

This prevents the browser from ever logging in and prevents me from being able to interact with the legacy projects page. 

The only way to get around is to turn off open links in app in Firefox. This should be more streamlined.

**Comment by stevepopovich** [comment]
Hi @majora2007 sorry to hear you are having issues. Am I correct in assuming you are using Android with the latest app store version?

**Reply by majora2007** [reply]
That is correct. Just updated yesterday to the latest.

**Reply by stevepopovich** [reply]
Do you not see the prompt in the app to link back to the browser? The flow for me is
1. Log into GitHub.com in Firefox
2. Goes to the GitHub app, shows a prompt to hit Firefox
3. Tap Firefox

Browser login is successful then. What part of this is failing? You can redirect back to the browser?

**Reply by stevepopovich** [reply, marked as answer]
Ok, yeah, this is def weird @majora2007 I filed with the team and I will be investigating this today. Thanks for writing. We will write back here when we have an update or a fix.

**Reply by manuel-rw** [reply]
Hi @stevepopovich , I'm also experiencing this issue with app version ``1.206.0``. It completely breaks GitHub Web for me on mobile and since the app isn't complete in its feature set and often behind the web app, I cannot use a lot of features. Let me know whether I can provide you with debug logs or additional information.

**Reply by levino** [reply]
I have the same issue. "Archiving" the github app was the only way to enable me to login into github from firefox. Happy to provide further information if helpful.

**Reply by beliaev-maksim** [reply]
Hit the same issue

Disabling Open links helped ![Screenshot_2025-07-07-17-57-20-821_com.android.settings.jpg](https://github.com/user-attachments/assets/b75a8fa1-0ee1-421f-bf32-74db19a041d5)

**Comment by klofroxrd** [comment]
response #2
